react-components/src/components/RuleBasedOutputs/types.ts

```diff
@@ -220,11 +220,6 @@ export type SimpleSource = {
   version: string;
 } & DmsUniqueIdentifier;
 
-export type DmsUniqueIdentifier = {
```

Comment:
Dms? What is that? I prefer proper names.

Reply:
It's not super-relevant to the PR, it's just a small cleanup. But DMS is the low-level API to the FDM-backend. Really not sure how to name this differently without being very verbose (also, I don't remember what the letters stand for)